scripts/repl.js

```diff
@@ -212,8 +212,9 @@
     }
   };
 
+  var capturingConsole;
```

Comment:
Should we just use the same instance every time and create the console object here?